tests/wpt/mozilla/tests/mozilla/resources/.gitignore

```diff
@@ -0,0 +1 @@
+resources.js
```

Comment:
What if you add an entry to the top-level .gitignore instead?

Reply:
Same thing. It continues to be added to the manifest